mail/mail-utils.go

```diff
@@ -0,0 +1,121 @@
+package mail
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"text/template"
+
+	"github.com/wopta/goworkspace/lib"
+	"github.com/wopta/goworkspace/models"
+)
+
+func CheckChannel(policy *models.Policy) string {
```

Comment:
rename function into `getChannel`.
In general this utils functions are not needed outside this domain so I think it's better to have the function name starting with lower case letter